lib/ui/splash/splash_page.dart

```diff
@@ -60,9 +63,9 @@ class _SplashPageState extends State<SplashPage> {
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      body: Center(
-        // This page shouldnt be visible for more than a split second, but if
+    return CobbleScaffold(
+      child: Center(
+        // This page shouldn't be visible for more than a split second, but if
```

Comment:
Is there a chance that this could produce annoying blinking for the user?

e.g. if this progress bar displays and then split second later it blinks out of the existence and real content displays in its place.

Reply:
It might flash (depending on how fast user's SharedPreferences are but this is getting reworked with new setup so I wouldn't like to fix it just because I renamed a component :)